lib/cc/engine/analyzers/php/main.rb

```diff
@@ -20,7 +20,7 @@ class Main < CC::Engine::Analyzers::Base
           private
 
           def process_file(path)
-            code = File.read(path)
+            code = File.binread(path)
```

Comment:
This doesn't seem related to the dir change. Is there something in PHP source or how we parse it that prompted this?